ops/travis/e2e-testing/utils/auto.js

```diff
@@ -278,6 +276,87 @@ async function dashboardDeleteFirstStudy(page) {
   await page.click('[osparc-test-id="confirmDeleteStudyBtn"]')
 }
 
+async function openNode(page, pos) {
+  console.log("Opening Last Node");
+
+  const children = await utils.getNodeTreeItemIDs(page);
+  console.log("children", children);
+  if (children.length < pos+1) {
+    console.log("Node tree items not found");
+    return;
+  }
+  const lastChildId = '[osparc-test-id="' + children[pos] + '"]';
+  await page.waitForSelector(lastChildId);
+  await page.click(lastChildId);
+
+  await page.waitForSelector('[osparc-test-id="openServiceBtn"]');
+  await page.click('[osparc-test-id="openServiceBtn"]');
+}
+
+async function openLastNode(page) {
+  console.log("Opening Last Node");
+
+  const children = await utils.getNodeTreeItemIDs(page);
+  console.log("children", children);
+  if (children.length < 1) {
+    console.log("Node tree items not found");
+    return;
+  }
+  const lastChildId = '[osparc-test-id="' + children.pop() + '"]';
+  await page.waitForSelector(lastChildId);
+  await page.click(lastChildId);
+
+  await page.waitForSelector('[osparc-test-id="openServiceBtn"]');
+  await page.click('[osparc-test-id="openServiceBtn"]');
+}
+
+async function checkDataProducedByNode(page) {
+  console.log("checking Data produced by Node")
+
+  await page.waitForSelector('[osparc-test-id="nodeViewFilesBtn"]')
+  await page.click('[osparc-test-id="nodeViewFilesBtn"]')
+
+  await page.waitFor(10000)
```

Comment:
Same here... Why 10s? What if it takes 12 seconds for whatever reason, or maybe it takes 2 seconds and we are wasting time. Is it possible that when the tree loads, it makes the test to continue?